.gitignore

```diff
@@ -19,3 +19,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+package-lock.json
```

Comment:
Why are you adding this to gitignore? You should commit this file and not ignoring it.